File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0) return false;
        int y=0;
        int z=x;
        while(z != 0){
            y = y* 10+z%10;
            z /=10;
        }
        if(y == x) return true;
        else return false;
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        num = str(x)
        for i in range(0, len(num)//2):
            if num[i] != num[-i-1]:
                return False
        return True
        